web/ViewModels/LandlordProfileModel.cs

```diff
@@ -86,5 +85,22 @@ public static LandlordProfileModel FromDbModel(LandlordDbModel landlord)
                 Postcode = landlord.Postcode
             }
         };
+
+        if (landlord.Title != "Mr"
+            && landlord.Title != "Mrs"
+            && landlord.Title != "Miss"
+            && landlord.Title != "Ms"
+            && landlord.Title != "Dr"
+            && landlord.Title != "Prof")

```

Comment:
We can simplify this slightly :) 

```suggestion
        if (new[] { "Mr", "Mrs", "Miss", "Ms", "Dr", "Prof" }.Contains(landlord.Title))

```